### PR TITLE
Gitlab Integration Changes for V1.4

### PR DESCRIPTION
**Pull request by r-pai**
Gitlab Integration Changes for V1.4

### CODE REVIEW

File: installscripts/dockerfiles/gitlab/gitlab.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+yes yes| /opt/gitlab/bin/gitlab-rake gitlab:setup RAILS_ENV=production GITLAB_ROOT_PASSWORD=gitlab12345 > ~/out.txt 2>&1
```
Comment (suryajak) [inline installscripts/dockerfiles/gitlab/gitlab.sh]: Why this password hardcoded? Use a random value.

File: installscripts/dockerfiles/gitlab/privatetoken.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import requests
+from urlparse import urljoin
+from bs4 import BeautifulSoup
+
+endpoint = u"http://localhost"
+root_route = urljoin(endpoint, u"/")
+sign_in_route = urljoin(endpoint, u"/users/sign_in")
+pat_route = urljoin(endpoint, u"/profile/personal_access_tokens")
+
+login = u"root"
```
Comment (suryajak) [inline installscripts/dockerfiles/gitlab/privatetoken.py]: Lets use JazzAdmin email address as the login. If email address isn't allowed, then userid based on email address.

File: installscripts/dockerfiles/gitlab/privatetoken.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+password = u"gitlab12345"
```
Comment (suryajak) [inline installscripts/dockerfiles/gitlab/privatetoken.py]: No hardcoded password

File: installscripts/dockerfiles/gitlab/gitlab.sh (hunk 1 above)
Comment (r-pai) [reply installscripts/dockerfiles/gitlab/gitlab.sh]: The hard coded password is removed .

File: installscripts/dockerfiles/gitlab/privatetoken.py (hunk 2 above)
Comment (r-pai) [reply installscripts/dockerfiles/gitlab/privatetoken.py]: I need to check about this. But I would suggest we have these as a separate ticket as these are would be an enhancement type.

File: installscripts/dockerfiles/gitlab/privatetoken.py (hunk 3 above)
Comment (r-pai) [reply installscripts/dockerfiles/gitlab/privatetoken.py]: Done.

**Pull request merged**